cluster-provision/k8s/1.25/manifests/kubeadm.conf

```diff
@@ -19,7 +19,11 @@ apiServer:
     audit-policy-file: /etc/kubernetes/audit/adv-audit.yaml
```

Comment:
Note please that also kubeadm_ipv6.conf need to be changed, for all providers

Reply:
Do we get error or is ipv6 still working?

Reply:
Yes, that sounds good. Here I just did the first step. It's very easy to pick this up but as I mentioned... first we need to enable Kubevirt

Reply:
honestly i don't see why it can't be done in parallel
just to align them all, and not forget it

but whatever you prefer

Reply:
I am not saying it can't be done in parallel. But I personally lack the bandwidth. What I suggested is that this becomes "required" when we fix Kubevirt and for now is only nice to have.